src/org/ods/OpenshiftUtils.groovy

```diff
@@ -0,0 +1,13 @@
+package org.ods
+
+class OpenshiftUtils {
+
+    static boolean environmentExists(def script, String name) {
+        def statusCode = script.sh(
+            script:"oc project ${name} &> /dev/null",
+            label: "check if OCP environment exists",
```

Comment:
michael - can you please add the `env` back into the label?

Reply:
Good point, I didn't even realize that the two versions of `enviornmentExists` differed and by chance I took the "wrong one" without the name in the label. Fixed in ef1c2f2b6ddd04fef05123392e4f41067e5f4d2b.